fhir-swagger-generator/src/main/java/com/ibm/fhir/openapi/generator/FHIROpenApiGenerator.java

```diff
@@ -1070,6 +1073,9 @@ private static void generateDefinition(Class<?> modelClass, JsonObjectBuilder de
                 JsonObjectBuilder wrapper = factory.createObjectBuilder();
                 wrapper.add("type", "object");
                 wrapper.add("properties", properties);
+                if (!requiredArray.isEmpty()) {
```

Comment:
if requiredArrary is used here only, better to move "JsonArray requiredArray = required.build();" to just above it.